File: Java/src/main/java/org/codecop/dependencies/d/RestCountriesAPIException.java
package org.codecop.dependencies.d;

public class RestCountriesAPIException extends RuntimeException {

    private static final long serialVersionUID = -7774153411692345736L;

    public RestCountriesAPIException(String message) {
        super(message);
    }

    public RestCountriesAPIException(String message, Throwable cause) {
        super(message, cause);
    }

}

File: CSharp/DependencyBreakingKatas/D/RestCountriesAPIException.cs
using System;

namespace Org.Codecop.Dependencies.D
{
    public class RestCountriesAPIException : ApplicationException
    {
        public RestCountriesAPIException(string message)
            : base(message)
        {
        }

        public RestCountriesAPIException(string message, Exception cause)
            : base(message, cause)
        {
        }
    }
}
